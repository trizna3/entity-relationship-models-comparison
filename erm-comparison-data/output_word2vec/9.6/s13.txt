contract-EMPTY
account-customer account
real estate types-EMPTY
life insurance-life insurance
real estate insurance-real estate insurance
persons;addresses-address
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estate data
CONTRACT_11_ASSOCIATION:E-persons-addresses
