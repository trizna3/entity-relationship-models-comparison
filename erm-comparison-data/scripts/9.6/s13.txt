entity set "customer account" {
	"name"
	"password"
};

entity set "insurance contract" {
	"date from"
	"date to"
	"fee"
};

entity set "life insurance" {
};

entity set "real estate insurance" {
};

entity set "insured data" {
	"insured"
	"death"
	"injury"
	"work incapacity"
};

entity set "real estate data" {
	"real estate"
	"crash"
	"elements"
	"address"
	"living area"
	"basement"
};

association "signs" between 
{
	"customer account" ONE 
	"insurance contract" MANY
}{
	"data"
};

association between 
{
	"life insurance" ONE 
	"insured data" MANY
}{};

association between 
{
	"real estate insurance" ONE 
	"real estate data" MANY
}{};

generalization from "real estate insurance" to "insurance contract";

generalization from "life insurance" to "insurance contract";
