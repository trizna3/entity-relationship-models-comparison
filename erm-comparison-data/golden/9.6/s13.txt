account-customer account
contract-insurance contract
life insurance-life insurance
real estate insurance-real estate insurance
persons-insured data
addresses-address
real estate types-EMPTY
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estate data
